debezium-server/debezium-server-core/src/test/java/io/debezium/server/DebeziumServerIT.java

```diff
@@ -66,4 +69,22 @@ public void testPostgresWithJson() throws Exception {
         Assertions.assertThat(((String) testConsumer.getValues().get(MESSAGE_COUNT - 1))).contains(
                 "\"after\":{\"id\":1004,\"first_name\":\"Anne\",\"last_name\":\"Kretchmar\",\"email\":\"annek@noanswer.org\"}");
     }
+
+    @Test
+    public void testDebeziumMetricsWithPostgres() {
+        Testing.Print.enable();
+
+        Awaitility.await().atMost(Duration.ofSeconds(TestConfigSource.waitForSeconds())).until(() -> {
+            try {
+                // snapshot process finished
+                // and consuming events finished!
+                return metrics.snapshotCompleted()
+                        && metrics.streamingQueueCurrentSize() == 0;
+            }
+            catch (Exception e) {
+                e.printStackTrace();
```

Comment:
Could you please use a logger here? Thanks

Reply:
removed the line